_studio/shared/include/mfx_config.h

```diff
@@ -88,6 +88,9 @@
     #if MFX_VERSION >= 1027
         #define MFX_ENABLE_H264_ROUNDING_OFFSET
     #endif
+    #if MFX_VERSION >= 1028
```

Comment:
Common way to introduce new functionality, which requires API change, is to put it under MFX_VERSION >= MFX_VERSION_NEXT. Thus anyone who build master w/o specifying -DAPI:STRING=latest would get only released API features. When feature considered ready,  MFX_VERSION_NEXT replaced by next API number.  For this one it would be at least 1.29, as API 1.28 finalized three months ago and published as Q3 release.

Reply:
Thanks for the explanation, I will modify them.